values.yaml

```diff
@@ -42,6 +42,10 @@ glueops_alerts:
   # -- Found at `opsgenie_credentials` in the json output that is part of `opsgenie_prometheus_api_keys` output from terraform-module-cloud-multy-prerequisites: https://github.com/GlueOps/terraform-module-cloud-multy-prerequisites
   opsgenie_apikey: placeholder_opsgenie_api_key
 
+glueops_cluster_monitoring:
+  opsgenie_apikey_cluster_monitoring: placeholder_opsgenie_api_key_cluster_monitoring
+  ping_sleep: 10
```

Comment:
is this in minutes? or seconds?

Reply:
If it's minutes could you do 3 minutes as the default? It may create alarms on cluster setup but afterwards it should be pretty fast. @fernandoataoldotcom any thoughts against this?

Reply:
seconds
do i add a comment saying the value of ping sleep is in seconds

Reply:
ya, changing the variable name to reflect the units would be good
I don't have a perspective on how long we should allow for no pings to be sent before we alert - I think the smallest increment available in opsgenie is minutes, so if we go for 1 min, we'd have to send the ping more frequently than once a minute.

Reply:
@Mbaoma I'd suggest ping_sleep_seconds (if it's seconds) and ping_sleep_ms (if it's miliseconds).

Also, please configure it to 2 minutes. I think 1 minute might be too fast but once every 2 minutes seems more reasonable and if we move pods around it should give us enough time to fire off another heartbeat before we get an alert.

Reply:
noted